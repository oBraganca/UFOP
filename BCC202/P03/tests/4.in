10
I Gabriel
505498 nivus 1 1980 Gabriel 209
110103 hb20 1 2010 Davi 207
683629 polo 2 2021 Pedro 206
246188 fox 1 2003 Sophia 202
351079 tcross 10 1988 Isabella 208
498503 yars 10 2001 Bernardo 205
892003 polo 6 2017 Arthur 203
986719 nivus 10 2016 Bernardo 205
563020 fox 6 1990 Arthur 203
561405 argo 9 1987 Sophia 202
