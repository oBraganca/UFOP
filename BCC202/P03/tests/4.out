nivus [1980|1|505498]
